pkg/render/manager.go

```diff
@@ -534,6 +534,9 @@ func (c *managerComponent) voltronContainer() corev1.Container {
 
 	if c.cfg.Tenant != nil {
 		env = append(env, corev1.EnvVar{Name: "VOLTRON_TENANT_ID", Value: c.cfg.Tenant.Spec.ID})
+		// Always configure the Tenant Claim for all multi-tenancy setups (single tenant and multi tenant)
+		// This will check the tenant claim when a Bearer token is presented to Voltron
+		env = append(env, corev1.EnvVar{Name: "VOLTRON_OIDC_REQUIRE_TENANT_CLAIM", Value: "true"})
```

Comment:
This is a nit, but I don't think this should say OIDC in the name. The tenant claim isn't one of the claims defined in the OIDC specification as far as I can tell: https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims

I think this should just be VOLTRON_REQUIRE_TENANT_CLAIM